examples/gpu/gpu_librett.cpp

```diff
@@ -23,7 +23,7 @@
 
 #ifdef TILEDARRAY_HAS_CUDA
```

Comment:
cmake variable names need to be updated too

Reply:
@evaleev If you agree, I'd like to limit the changes in Step 1 (this PR) to directory and file suffixes. Step 2 will deal with .cu -> .cpp conversion. Step 3 will address the changes in preprocessor definitions and CMake variables. By splitting the entire process into three consecutive steps we can group the changes by the topic, narrow down the discussions, and achieve faster overall progress. Should we stay with this plan or revise it?